packages/lit-html/src/test/test-utils/memory.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2017 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+declare global {
+  interface Window {
+    gc: () => void;
+  }
+  interface Performance {
+    memory: {
+      usedJSHeapSize: number;
+    };
+  }
+}
+
+const canRunMemoryTests =
+  globalThis.performance?.memory?.usedJSHeapSize && window.gc;
+
+export const memorySuite = canRunMemoryTests ? suite : suite.skip;
+
+export const forceGC = () => {
+  window.gc();
```

Comment:
Do you know of any links that talk about using window.gc() in tests? Is 10 calls the most reliable way to do it? Would waiting a tick help?

Reply:
Not really. Calling it more than once is cargo-culted from somewhere, including that calling it in a loop doesn't have the same effect; I can't recall where from.

Reply:
Let's at least add a comment.